util/gh-pages/index.html

```diff
@@ -180,6 +181,22 @@ <h4 class="list-group-item-heading">
             }
         }
 
+        function searchLint(lint, therm) {
```

Comment:
Thanks for pulling this out. As an aside, I think `therm` should be `term` here and below?

Reply:
Spelling is not always my strong suite 😅 thanks for pointing that out. It should be fixed now 🙃